src/components/Notifications.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div id="oc-notification" class="uk-navbar-item">
+    <oc-icon id="oc-notification-bell" name="info" variation="inverted" ariaLabel="Notifications" />
+    <oc-drop id="oc-notification-drop" toggle="#oc-notification-bell" boundary="#oc-topbar" class="uk-width-large">
+      <div v-for="(el, index) in activeNotifications" :key="index" class="uk-width-1-1 uk-background-muted uk-padding-small uk-display-inline-block">
```

Comment:
I would drop the muted background and add horizontal line to separate them. I think it looks better IMHO.